packages/devtools_app/lib/src/flutter/screen.dart

```diff
@@ -19,3 +30,72 @@ abstract class Screen {
   /// Builds the body to display for this tab.
   Widget build(BuildContext context);
 }
+
+enum DevToolsScreenType {
+  inspector,
+  timeline,
+  memory,
+  performance,
+  logging,
+  info,
+  connect,
+  debugger,
+  network,
+  simple,
```

Comment:
nit: rename simple to
placeholder or testOnly 
or something that makes it clear why it exists.

Reply:
simple refers to the SimpleScreen class. This is used for the connect screen as well as for URI not found.